### PR TITLE
cleanup warnings

### PR DESCRIPTION
**Pull request by jframe**
Cleanup some warnings and code. Mostly using method references where possible and final.

### CODE REVIEW

File: dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java [hunk 1]
```diff
@@ -63,8 +63,7 @@ public boolean equals(Object obj) {
     if (getClass() != obj.getClass()) return false;
     GenesisAddress other = (GenesisAddress) obj;
     if (hex == null) {
-      if (other.hex != null) return false;
-    } else if (!hex.equals(other.hex)) return false;
-    return true;
+      return other.hex == null;
+    } else return hex.equals(other.hex);
```
Comment (usmansaleem) [inline dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java:67]: nit: `else {
      return ... ;
      }`
use brackets for else condition.

**Review by usmansaleem — APPROVED** [review]
LGTM - one minor suggestion around else block.

File: dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java (hunk 1 above)
Comment (jframe) [reply dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java:67]: done. reverted that change. didn't mean to mess with the equals

File: dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java [hunk 2]
```diff
@@ -63,7 +63,8 @@ public boolean equals(Object obj) {
     if (getClass() != obj.getClass()) return false;
     GenesisAddress other = (GenesisAddress) obj;
     if (hex == null) {
-      return other.hex == null;
-    } else return hex.equals(other.hex);
+      if (other.hex != null) return false;
+    } else if (!hex.equals(other.hex)) return false;
+    return true;
```
Comment (usmansaleem) [inline dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java]: oh is this auto-generated equals method?

**Review by usmansaleem — APPROVED** [review]
LGTM

**Pull request merged**